packages/itwinui-react/src/core/List/List.test.tsx

```diff
@@ -0,0 +1,94 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { List } from './List';
+import { ListItem } from './ListItem';
+
+it('should render in its most basic state', () => {
+  const { container } = render(
+    <List>
+      <ListItem>item 1</ListItem>
+      <ListItem>item 2</ListItem>
+      <ListItem>item 3</ListItem>
+    </List>,
+  );
+  expect(screen.getByRole('list')).toHaveClass('iui-list');
+  const items = container.querySelectorAll('.iui-list-item');
+  expect(items).toHaveLength(3);
+  items.forEach((item, index) => {
+    expect(item).toHaveTextContent(`item ${index + 1}`);
+  });
+});
+
+describe('ListItem', () => {
```

Comment:
It'll probably be easier for the future if the 'ListItem' tests are in their own file.

Reply:
moved